core/validation/utils.py

```diff
@@ -12,3 +19,48 @@ def remove_duplicate_indexes(df: pd.DataFrame):
     :return: the DataFrame without duplicate indexes
     """
     return df[~df.index.duplicated(keep="first")]
+
+
+def get_cell_indexes_for_outcomes(failed_row: pd.Series, index: int) -> str:
+    """
+    Constructs cell indexes for outcomes based on the provided failed row and index.
+
+    The function determines whether the error occurred in footfall outcomes (starting from row 60)
+    or non-footfall outcomes. If the index is greater than or equal to 60, it calculates the cell
+    index for footfall outcomes based on the provided 'Start_Date'. If the index is less than 60,
+    it calculates the cell index for non-footfall outcomes.
+
+    For footfall outcomes, the cell index is determined by adding a row index gap calculated from
+    the start year of the UK financial year as each year represents an additional 5 rows in the
+    original spreadsheet from the row where the entry for a given footfall outcome's data begins.
+
+    :param failed_row: A pandas Series representing a row where an error has occurred.
+    :param index: The index indicating the position of the error in the data.
+    :return: A string containing the constructed cell index for outcomes.
+    """
+    start_date = failed_row["Start_Date"]
+    financial_year = get_uk_financial_year_start(start_date)
```

Comment:
I thought `start_date` was a string? Or now this is a series does it return a datetime?

Reply:
It's a datetime, converted to a string before as I believed that would be easier for manipulating it. Now we can just leave as it is